leptos_async_signal/Cargo.toml

```diff
@@ -1,10 +1,16 @@
 [package]
 name = "leptos_async_signal"
+authors = ["Demiurg <contact@demiurg.hr>"]
```

Comment:
The e-mail address should be <developers@demiurg.hr>.

Reply:
Done, thanks!